test/tests/expected/global_deps_slim.js

```diff
@@ -0,0 +1,8 @@
+[
+    global,
+    function (stealRequire, stealExports) {
+        stealRequire(1);
+        stealRequire(2);
+        var GLOBAL = 'I don\'t like "Quotes"';
```

Comment:
oh, you mean we should transform `window.[PROP]` to `stealExports[PROP]` or should we require users to correctly configure metadata for cases like that?